boot/zephyr/CMakeLists.txt

```diff
@@ -13,16 +13,16 @@ set(BOARD qemu_x86)
 # and fits inside, the boot partition. (If the user specified a
 # DTC_OVERLAY_FILE on the CMake command line, we need to append onto
```

Comment:
Should we state `${IMAGE}DTC_OVERLAY_FILE` or even `mcuboot_DTC_OVERLAY_FILE` ?

IMO it is not needed as how to set this variable is documented elsewhere, and here we just need the motivation for the code.

Reply:
Agreed.